### DISCUSSION TITLE
Use the result type "System.Collections.IEnumerable"?

### DISCUSSION THREAD

**Discussion by pri-joao-martins** [JasperFx/marten#2537] [Q&A]
I'm using the [Community.OData.Linq](https://github.com/IharYakimush/comminity-data-odata-linq) library, but when trying to perform the OData filter in an `IMartenQueryable` the following error is thrown:

`Marten does not know how to use result type System.Collections.IEnumerable`

Any workaround I can apply here?

Thank you.

**Comment by jeremydmiller** [comment]
Hey, can you add a little more detail here?
